app.js

```diff
@@ -0,0 +1,13 @@
+const express = require("express");
+const { getTopics } = require("./Controllers/api.controller");
+
+const app = express();
+app.use(express.json());
```

Comment:
don't really need line 5 until we have reason to parse something on the incoming request (e.g. POST/PATCH/params...)